cluster/gce/util.sh

```diff
@@ -2742,13 +2742,18 @@ function create-network() {
       --allow "tcp:1-2379,tcp:2382-65535,udp:1-65535,icmp" &
   fi
 
+  NODE_TAGS="${NODE_TAG}"
```

Comment:
Adding some comments here could help code review ( for the PR or for future reads)